schemas/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "federated-api-model-schemas",
```

Comment:
I'm assuming this package is not intended to be published in any registry. If that's correct I would suggest adding a `private: true` (see https://docs.npmjs.com/cli/v6/configuring-npm/package-json#private)

Reply:
That's a fair shout - I think right now it's going to be private, but longer term this could be something packaged as a project.

I'll make it `private` :+1: